solr_plugins/MultiLanguageQueryParser/src/de/uni_tuebingen/ub/ixTheo/MultiLanguageQueryParser.java

```diff
@@ -229,6 +230,10 @@ else if (subquery instanceof PrefixQuery) {
             subquery = processPrefixQuery((PrefixQuery)subquery);
             return new BoostQuery(subquery, queryCandidate.getBoost());
         }
+        else if (subquery instanceof PhraseQuery) {
```

Comment:
The "else if" and the "else" below should cuddle up to the closing braces above.